cmd/cli/root.go

```diff
@@ -0,0 +1,94 @@
+package cli
+
+import (
+	"fmt"
+	"os"
+
+	mcobra "github.com/muesli/mango-cobra"
+	"github.com/muesli/roff"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+
+	"github.com/bom-squad/go-cli/cmd/cli/options"
+	"github.com/bom-squad/go-cli/pkg/log"
+)
+
+var version = "dev"
+
+func ManCommand(root *cobra.Command) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "man",
+		Short:                 "Generates command line manpages",
+		SilenceUsage:          true,
+		DisableFlagsInUseLine: true,
+		Hidden:                true,
+		Args:                  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			manPage, err := mcobra.NewManPage(1, root)
+			if err != nil {
+				return err
+			}
+
+			_, err = fmt.Fprint(os.Stdout, manPage.Build(roff.NewDocument()))
+			return err
+		},
+	}
+
+	return cmd
+}
+
+func NewRootCmd() *cobra.Command {
+	ro := &options.RootOptions{}
+	rootCmd := &cobra.Command{
+		Use:     "sbom-convert",
+		Version: version,
+		Short:   "",
+		Long:    ``,
```

Comment:
Should we add a minimum required arguments?
Also place holder comment to Fill in the long, short and examples field at some point .

Reply:
`Args:              cobra.ExactArgs(1),` 

Yes, adding short/long placeholder + examples.